pkg/plugins/runtime/k8s/controllers/gateway_instance_controller.go

```diff
@@ -101,18 +106,15 @@ func k8sSelector(name string) map[string]string {
 	return map[string]string{"app": name}
 }
 
-func (r *GatewayInstanceReconciler) resolveGatewayInstanceServiceTag(gatewayInstance *mesh_k8s.MeshGatewayInstance) {
+func (_ *GatewayInstanceReconciler) gatewayInstanceTags(gatewayInstance *mesh_k8s.MeshGatewayInstance) map[string]string {
```

Comment:
Why not changing it to regular function?

Reply:
controllers is quite a big package. I wanted to keep the scope of a function on the reconciller only. I can change it if you think it's important

Reply:
No, I don't think it's necessary